huntflow_api_client/entities/applicants.py

```diff
@@ -123,3 +126,147 @@ async def delete(self, account_id: int, applicant_id: int) -> None:
             "DELETE",
             f"/accounts/{account_id}/applicants" f"/{applicant_id}",
         )
+
+    async def search(
+        self,
+        account_id: int,
+        q: Optional[str] = None,
+        tag: Optional[List[int]] = None,
+        status: Optional[List[int]] = None,
+        rejection_reason: Optional[List[int]] = None,
+        vacancy: Union[List[int], str, None] = None,
+        account_source: Optional[List[int]] = None,
+        only_current_status: bool = False,
+        field: ApplicantSearchField = ApplicantSearchField.all,
+        count: int = 30,
+        page: int = 1,
+    ) -> ApplicantSearchResponse:
+        """
+        API method reference:
+            https://api.huntflow.ai/v2/docs#get-/accounts/-account_id-/applicants/search
+
+        :param account_id: Organization ID
+        :param q: Search query
+        :param tag: List of tag ID
+        :param status: List of vacancy status ID
+        :param rejection_reason: List of rejection reason ID
+        :param vacancy: List of vacancy ID or 'null', If you pass a 'null' value,
+            then applicants who have not been added to any vacancy will be displayed
```

Comment:
I'd prefer to not use `null` at all here. It looks redundant in pythonic interface, because you can achieve the desired behavior without it.
* `vacancy = [1,2,3]` - applicants assigned to specified vacancies
* `vacancy = None` - no filter for vacancies
* `vacancy = []` - empty list means applicant is not assigned to any vacancy